user.js

```diff
@@ -1153,14 +1153,17 @@ user_pref("_user.js.parrot", "9100 syntax error: this parrot is blind!");
  * if the user wishes to keep the existence of the mail provider private.
  * We also enforce (valid) SSL/TLS connections if auto-configuration happens to be enabled.
  * [1] https://developer.mozilla.org/en-US/docs/Mozilla/Thunderbird/Autoconfiguration ***/
-user_pref("mailnews.auto_config.guess.enabled", false);
+// user_pref("mailnews.auto_config.guess.enabled", false);
```

Comment:
I'm not sure about keeping autoconfig enabled, **by default**.
This project is a template, aiming to provide an hardened version of Thunderbird, and autoconfig stays one of these features explicitly _leaking_ future usages information to third-parties (even if they don't have malicious intents, their servers receive such data) like Mozilla or ISP.
If we want TB to "simply" speaks IMAP/POP/SMTP, we can't expect it to send mail server addresses _away_ beforehand.

When we use a giant email providers (like Google or Hotmail), it's not necessarily an issue (from a privacy point of view here), but if your email server is somehow specific and maybe not public (self-hosted, proposed by company, or maybe a Tor hidden service ?), I guess this becomes problematic.

Reply:
yes for sure. My idea was to add commented out lines for more security and privacy if the ISP would be used (for whatever reason).

But to my knowledge autoconfig is used to get the config from the mail provider, so the request is sent to their servers, not some random others.

If this is not true, many changes can be reverted, as contacting random 3rd parties makes little sense

Reply:
> But to my knowledge autoconfig is used to get the config from the mail provider, so the request is sent to their servers, not some random others.

Unfortunately it doesn't work that way : a query is sent to Mozilla servers to gather information about mail providers own servers (cf. `mailnews.auto_config_url` value).

> If this is not true, many changes can be reverted, as contacting random 3rd parties makes little sense

I am okay with other changes, as long as we keep AC disabled by default :wink: 

PS : I've noticed there is an explicit AC mention in script top comment, pointing to `9101` and `9102` re-enabling if needed.

Reply:
So I've reverted this specific part according to above point.